src/components/Activity/Activity.js

```diff
@@ -12,6 +12,7 @@ import { dimensions } from '../Shared/Dimensions';
 import Connected from './Connected';
 import NewRequests from './NewRequests';
 import ScrollableTabView from 'react-native-scrollable-tab-view';
+import DefaultTabBar from './DefaultTabBar';
```

Comment:
Alphabetical ordering is not in style guide isn't it?

I could confirm style error on line 14. The error is
> Absolute imports should come before relative imports
I found same thing in several files but I have not mentioned it.